modules/cij/__init__.py

```diff
@@ -52,43 +52,46 @@ def get_time_stamp():
     return ""
 
 
-def info(txt):
+def info(txt, fd=sys.stdout):
```

Comment:
Put these changes to the output functions into their own commit describing what they enable

Reply:
Roger!